examples/private_information_retrieval/database.xml

```diff
@@ -0,0 +1 @@
true</installed><locked>false</locked><installDate>1389094260000</installDate><removalDate></removalDate><temporary>false</temporary><nbBikes>14</nbBikes><nbEmptyDocks>8</nbEmptyDocks><nbDocks>24</nbDocks></station><station><id>758</id><name>Westbourne Park Road, Portobello</name><terminalName>300057</terminalName><lat>51.51787005</lat><long>-0.201005397</long><installed>true</installed><locked>false</locked><installDate>1389178020000</installDate><removalDate></removalDate><temporary>false</temporary><nbBikes>12</nbBikes><nbEmptyDocks>9</nbEmptyDocks><nbDocks>23</nbDocks></station><station><id>759</id><name>Broadley Terrace, Marylebone</name><terminalName>300072</terminalName><lat>51.52456169</lat><long>-0.165668686</long><installed>true</installed><locked>false</locked><installDate>1389627480000</installDate><removalDate></removalDate><temporary>false</temporary><nbBikes>3</nbBikes><nbEmptyDocks>15</nbEmptyDocks><nbDocks>18</nbDocks></station><station><id>760</id><name>Rossmore Road, Marylebone</name><terminalName>300071</terminalName><lat>51.52526975</lat><long>-0.163795009</long><installed>true</installed><locked>false</locked><installDate>1390310760000</installDate><removalDate></removalDate><temporary>false</temporary><nbBikes>12</nbBikes><nbEmptyDocks>6</nbEmptyDocks><nbDocks>18</nbDocks></station><station><id>761</id><name>Humbolt Road, Fulham</name><terminalName>300068</terminalName><lat>51.48321729</lat><long>-0.211860644</long><installed>true</installed><locked>false</locked><installDate>1389781800000</installDate><removalDate></removalDate><temporary>false</temporary><nbBikes>14</nbBikes><nbEmptyDocks>12</nbEmptyDocks><nbDocks>26</nbDocks></station><station><id>762</id><name>Storey's Gate, Westminster</name><terminalName>200202</terminalName><lat>51.50070305</lat><long>-0.129698963</long><installed>true</installed><locked>false</locked><installDate>1392039000000</installDate><removalDate></removalDate><temporary>false</temporary><nbBikes>14</nbBikes><nbEmptyDocks>5</nbEmptyDocks><nbDocks>21</nbDocks></station><station><id>763</id><name>Mile End Park Leisure Centre, Mile End</name><terminalName>300027</terminalName><lat>51.52059714</lat><long>-0.032566533</long><installed>true</installed><locked>false</locked><installDate>1392043020000</installDate><removalDate></removalDate><temporary>false</temporary><nbBikes>34</nbBikes><nbEmptyDocks>12</nbEmptyDocks><nbDocks>47</nbDocks></station><station><id>764</id><name>St. John's Road, Clapham Junction</name><terminalName>200185</terminalName><lat>51.46239255</lat><long>-0.16829214</long><installed>true</installed><locked>false</locked><installDate>1392633480000</installDate><removalDate></removalDate><temporary>false</temporary><nbBikes>29</nbBikes><nbEmptyDocks>4</nbEmptyDocks><nbDocks>33</nbDocks></station><station><id>765</id><name>Ranelagh Gardens, Fulham</name><terminalName>300064</terminalName><lat>51.46760141</lat><long>-0.20682737</long><installed>true</installed><locked>false</locked>
```

Comment:
Can you add a README file with a description of this file, its provenance, a license if needed, and some instructions on how to re-generate it when needed?